### ISSUE TITLE
False parameter after @name

### ISSUE THREAD

**Issue by Crowley1970** [Sedeniono/VSDoxyHighlighter#1]
We use the @name command pretty often to group methods under a common name, e.g.

```
/// @name Simple getter and setter
/// @{
void setFoo(int foo);
int getFoo() const;
/// @}

```
The problem is that "Simple" is interpreted as parameter and is colored accordingly to what is defined in Options -> Font and Colors. The text after @name should be interpreted as plain text.

**Comment by Sedeniono** [comment]
Huh, indeed, I misinterpreted the documentation for `@name`: I though a single word is supposed to follow.

But rather than interpreting everything after `@name` as ordinary comment text, it should get interpreted as title, right? In other words, `@name` should behave the same as [`@par`](https://www.doxygen.nl/manual/commands.html#cmdpar) or [`@mainpage`](https://www.doxygen.nl/manual/commands.html#cmdmainpage) regarding colorization, right? That should be easy to fix, but I can do so only next week.

**Comment by Crowley1970** [comment]
Thank you for your reply. Yes, from my point of view it would be great if @name was interpreted as title and thus behaved the same as @par and @mainpage.

**Comment by Sedeniono** [comment]
I fixed the issue in version 1.1.1 (available on [github](https://github.com/Sedeniono/VSDoxyHighlighter/releases/tag/v1.1.1) and also the [Visual Studio marketplace](https://marketplace.visualstudio.com/items?itemName=Sedenion.VSDoxyHighlighter)). Thanks again for the report!
